packages/o/opensnitch/files/correct-systemd-service.patch

```diff
@@ -0,0 +1,12 @@
+--- dev/null
++++ b/daemon/opensnitchd.service	2025-01-12 14:26:40.170570544 -0600
+@@ -5,8 +5,7 @@
+ [Service]
+ Type=simple
+ PermissionsStartOnly=true
+-ExecStartPre=/bin/mkdir -p /etc/opensnitchd/rules
+-ExecStart=/usr/local/bin/opensnitchd -rules-path /etc/opensnitchd/rules
++ExecStart=/usr/bin/opensnitchd -rules-path /etc/opensnitchd/rules
```

Comment:
Is this the correct rules path we want?

Reply:
It should be because the rules are created dynamically. Opensnitch does not provide any to build.